pkg/configtx/config.go

```diff
@@ -317,6 +317,10 @@ func setValue(cg *cb.ConfigGroup, value *standardConfigValue, modPolicy string)
 		return fmt.Errorf("marshaling standard config value '%s': %v", value.key, err)
 	}
 
+	if cg.Values == nil {
+		cg.Values = map[string]*cb.ConfigValue{}
+	}
```

Comment:
This was one oddity related to switching to New() for the config. It has to do with working with the UpdatedConfig() as derived by using proto.Clone on the original config. Empty maps are converted to nil and the code below would panic without this.